build_centos.dockerfile

```diff
@@ -0,0 +1,46 @@
+# https://github.com/ghdl/ghdl/issues/1484
+# https://github.com/ghdl/ghdl/issues/1118
+
+#
+# Centos 7
+#
+
+FROM centos:centos7
+
+RUN yum update -y && yum install -y git curl bzip2 wget flex fontconfig libX11 zlib-devel
+RUN yum install -y centos-release-scl && yum install -y devtoolset-8 texinfo
+SHELL [ "/usr/bin/scl", "enable", "devtoolset-8" ]
+
+#
+# Centos 8
+#
+
+#FROM centos:centos8
+
+#RUN yum update -y && yum install -y  git curl bzip2 wget flex fontconfig libX11 zlib-devel
+#RUN yum install -y --enablerepo=powertools gcc texinfo make
+
+#
+# Common
+#
+
+WORKDIR /root
+
+RUN mkdir gnat && \
+    curl -L https://community.download.adacore.com/v1/9682e2e1f2f232ce03fe21d77b14c37a0de5649b?filename=gnat-gpl-2017-x86_64-linux-bin.tar.gz | tar -xz -C gnat --strip-components=1
+RUN cd gnat && make ins-all prefix="build"
+
+ENV PATH=/root/gnat/build/bin:$PATH
```

Comment:
I suggest installing it to a standard location. A different user will probably have permission issues for executing from `/root/*`.